File: Java/4.19.java
/*

Many user-created passwords are simple and easy to guess. Write a program that takes a simple password and makes it stronger by replacing characters using the key below, and by appending "q*s" to the end of the input string.

i becomes !
a becomes @
m becomes M
B becomes 8
o becomes .
Ex: If the input is:

mypassword
the output is:

Myp@ssw.rdq*s
*/
import java.util.Scanner;

public class LabProgram {
   public static void main(String[] args) {
      /* Type your code here. */
      Scanner in = new Scanner(System.in);
      String s = in.nextLine();
      String newPassword = "";
      
         for (int i = 0; i < s.length(); ++i) {
            if(s.charAt(i)=='i')
               newPassword += "!";
            else if(s.charAt(i)=='a')
               newPassword += "@";
            else if(s.charAt(i)=='m')
               newPassword += "M";
            else if(s.charAt(i)=='B')
               newPassword += "8";
            else if(s.charAt(i)=='o')
               newPassword += ".";
            else
               newPassword += s.charAt(i);
        }
        
        newPassword += "q*s";
        System.out.println(newPassword);
   }
}

File: CPP/4.19.cpp
/*
Many user-created passwords are simple and easy to guess. Write a program that takes a simple password and makes it stronger by replacing characters using the key below, and by appending "!" to the end of the input string.

i becomes 1
a becomes @
m becomes M
B becomes 8
s becomes $
Ex: If the input is:

mypassword
the output is:

Myp@$$word!
*/

#include <iostream>
#include <string>
using namespace std;

int main() {
   
   /* Type your code here. */
   string userPassword;
   cin >> userPassword;
   
   for (int i = 0; i < userPassword.length(); i++) {
      if (userPassword[i] == 'i') {
         userPassword[i] = '1';
      }
      if (userPassword[i] == 'a') {
         userPassword[i] = '@';
      }
      if (userPassword[i] == 'm') {
         userPassword[i] = 'M';
      }
      if (userPassword[i] == 'B') {
         userPassword[i] = '8';
      }
      if (userPassword[i] == 's') {
         userPassword[i] = '$';
      }
   }
   
   printf("%s!\n", userPassword.c_str());

   return 0;
}
